membership-attribute-service/app/components/TouchpointBackends.scala

```diff
@@ -19,9 +19,9 @@ class TouchpointBackends(
     createMetrics: CreateMetrics,
     supporterProductDataServiceOverride: Option[SupporterProductDataService] = None,
     contactRepositoryOverride: Option[ContactRepository] = None,
-    subscriptionServiceOverride: Option[SubscriptionService] = None,
+    subscriptionServiceOverride: Option[SubscriptionService[Future]] = None,
     zuoraRestServiceOverride: Option[ZuoraRestService] = None,
-    catalogServiceOverride: Option[CatalogService] = None,
+    catalogServiceOverride: Option[CatalogService[Future]] = None,
```

Comment:
we had at some point got rid of the Future type parameter as it was only needed for the tests.  I have brought back the tests with minimal changes (for now) so we need the type parameters back again.

Reply:
😢

Reply:
of course we can get rid of them by making the tests use Future as well just like the main code (rather than just using unwrapped values) but I wasn't prepared to do it in this PR